packages/client/foreman_ygg_worker/foreman_ygg_worker.spec

```diff
@@ -5,19 +5,15 @@
 %global repo_orgname theforeman
 %global repo_name foreman_ygg_worker
 %global yggdrasil_libexecdir %{_libexecdir}/yggdrasil
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-%global yggdrasil_worker_conf_dir %{_root_sysconfdir}/yggdrasil/workers
+%global yggdrasil_worker_conf_dir %{_sysconfdir}/yggdrasil/workers
 
 %global goipath         github.com/%{repo_orgname}/%{repo_name}
 
-%if 0%{?rhel} > 7 && ! 0%{?fedora}
-%define gobuild(o:) \
-        GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback libtrust_openssl ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -compressdwarf=false -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v %{?**};
-%else
 %if ! 0%{?gobuild:1}
 %if 0%{?suse_version}
 %define gobuild(o:) GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '-Wl,-z,relro -Wl,-z,now '" -a -v %{?**};
```

Comment:
https://en.opensuse.org/openSUSE:Packaging_Go#Old_style_golang_applications has an example that requires `golang-packaging` and then they use `gobuild` from that but when I try it out on a SLES image I don't see the package.

Reply:
I see it on opensuse/leap:15.6 . Is that good enough or are we targeting sles specifically?

Reply:
We have that for ATIX so someone from @theforeman/atix should probably weigh in.

Reply:
I'm having a hard time figuring out how to do it right for suse. There seems to be a `gobuild` macro, but semantics of its use seem to be different from what we have in EL*. Unless anyone knows how to make this work, I'd leave it alone for now.